packages/terra-demographics-banner/src/_LargeDemographicsBannerDisplay.jsx

```diff
@@ -46,31 +56,46 @@ export default (props) => {
   delete customProps.className;
 
   return (
-    <section className={mainClasses} {...customProps}>
+    <section
+      {...customProps}
+      className={mainClasses}
+      tabIndex={-1}
```

Comment:
What is the reasoning behind this tabIndex? Do we have it for a11y as an additional stopping point for a screenreader?

Reply:
Yes, this is to prevent the entire banner from getting focus and only put focus on the Info tile.